Automattic/wp-calypso
# FSE: Rename BPB to Newspack blocks

#### Changes proposed in this Pull Request

* Does most of the renaming of things.

#### Testing instructions



* See #41423 for extensive testing instructions
* Sync command has changed to `$ yarn config set branch master && yarn run sync:newspack-blocks`
